fontforge/parsettf.c

```diff
@@ -3999,7 +3999,7 @@ return( 0 );
 	    SCConvertToOrder2(info->chars[i]);
     }
 
-    for ( i=0; fontnames[i]!=NULL && i<1; ++i ) {
+    for ( i=0; i<1 && fontnames[i]!=NULL ; ++i ) {
```

Comment:
Sorry, I don't know how to update a patch with Fontforge, would it be possible you merge it then just change this part?